src/common/LocIdCell.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { UUID } from '../logion-chain/UUID';
+import { Cell } from './Table';
+
+export interface Props {
+    id: string
+}
+
+export default function LocIdCell(props: Props) {
+
+    return (
+        <Cell
+            content={ new UUID(props.id).toDecimalString() }
```

Comment:
Why is it displayed in decimal (`256213782153833876180077727052610916945`) rather than usual hex notation (`c0c0fac1-eb68-4bca-8e67-4240371fd251`)

Reply:
Because only the decimal form is usable when querying directly the chain using Polkadot JS app.